docs/how-to/backup-your-databases.md

```diff
@@ -97,7 +103,8 @@ hooks:
 ```
 
 Note that you may need to use a `username` of the `postgres` superuser for
-this to work with PostgreSQL.
+this to work with PostgreSQL. Also, the `all` database name is not supported
+for SQLite databases.
```

Comment:
That's a fair omission! Do you think it's worth documenting what happens if `all` is used for a SQLite database.. Or even coding in an error if the user tries to set a name of `all`?

Reply:
Maybe a message with `logger.info` is enough, letting the user know it's treated as a general name for sqlite databases instead of a special keyword.

Reply:
Sounds good!